src/daos/Airtable/StudentDao.ts

```diff
@@ -0,0 +1,83 @@
+import Airtable, { FieldSet, Records } from "airtable";
+
+class StudentDao {
+  base_id = "appvMqcwCQosrsHhM";
+  api_key = process.env.AT_API_KEY;
+  airtable = new Airtable({ apiKey: this.api_key }).base(this.base_id);
```

Comment:
Do we need a `constructor` here to follow the same convention found in the `CanvasCoursesDao`

Reply:
[Done](https://github.com/Lambda-School-Labs/labs-be/pull/14/commits/cfbc7d8903736c135c64e17cb17142e007d5a9b9)